org.telegram.desktop.json

```
@@ -124,6 +127,122 @@
                 }
             ]
         },
+        {
+            "name": "qtbase",
+            "config-opts": [
+                "-archdatadir", "/app/lib/",
+                "-release",
+                "-opensource",
+                "-confirm-license",
+                "-shared",
+                "-no-separate-debug-info",
+                "-no-strip",
+                "-no-rpath",
+                "-optimized-qmake",
+                "-nomake", "examples",
+                "-nomake", "tests",
+                "-system-harfbuzz",
+                "-system-libjpeg",
+                "-system-libpng",
+                "-system-sqlite",
+                "-system-zlib",
+                "-system-proxies",
+                "-dbus-linked",
+                "-openssl-linked",
+                "-fontconfig",
+                "-accessibility",
+                "-glib",
+                "-icu",
+                "-no-directfb",
+                "-no-linuxfb",
+                "-no-kms"
```

Syscalls from new kernels should be avoided
```suggestion
                "-no-kms",
                "-no-feature-getentropy",
                "-no-feature-renameat2",
                "-no-feature-statx"
```